### PR TITLE
replace `df|DF` abbreviations with `dt|DT`

### PR DESCRIPTION
**Pull request by cjprybol**
no .pdf files or `findfirst` calls were harmed in the making of this pull request.

### CODE REVIEW

**Comment by cjprybol** [comment]
I branched this from https://github.com/JuliaData/DataTables.jl/pull/6 so this may need to wait until that one is accepted, or this can be rebased depending on the feedback

File: src/subdatatable/subdatatable.jl [hunk 1]
```diff
@@ -75,32 +75,34 @@ function SubDataTable{S <: Integer}(parent::DataTable, rows::AbstractVector{S})
 end
 
 
-function Base.sub{S <: Real}(dt::DataTable, rowinds::AbstractVector{S})
+=======
```
Comment (nalimilan) [inline src/subdatatable/subdatatable.jl]: Merging mistake here.

File: src/subdatatable/subdatatable.jl [hunk 2]
```diff
@@ -75,32 +75,34 @@ function SubDataTable{S <: Integer}(parent::DataTable, rows::AbstractVector{S})
...
-    return sub(adt, getindex(SimpleIndex(size(adt, 1)), rowinds))
+function Base.view(adt::AbstractDataTable, rowinds::Any)
+    return view(adt, getindex(SimpleIndex(size(adt, 1)), rowinds))
 end
 
-function Base.sub(adt::AbstractDataTable, rowinds::Any, colinds::Any)
-    return sub(adt[[colinds]], rowinds)
+function Base.view(adt::AbstractDataTable, rowinds::Any, colinds::Any)
+    return view(adt[[colinds]], rowinds)
+>>>>>>> master
```
Comment (nalimilan) [inline src/subdatatable/subdatatable.jl]: And here.

**Comment by nalimilan** [comment]
We should really have merged this immediately, before all the churn due to fixing deprecations... Sorry about that!

File: src/subdatatable/subdatatable.jl (hunk 1 above)
Comment (cjprybol) [reply src/subdatatable/subdatatable.jl]: thanks!

**Comment by nalimilan** [comment]
I've fixed a few more things, merging before more conflicts are added. Thanks!

**Comment by cjprybol** [comment]
Thanks for finishing this off @nalimilan!

File: src/abstractdatatable/abstractdatatable.jl [hunk 3]
```diff
@@ -673,34 +673,34 @@ without(df::AbstractDataTable, c::Any) = without(df, index(df)[c])
...
+Base.vcat(dt::AbstractDataTable) = dt
 
-Base.vcat(dfs::AbstractDataTable...) = vcat(AbstractDataTable[dfs...])
+Base.vcat(dts::AbstractDataTable...) = vcat(AbstractDataTable[dts...])
 
-Base.vcat(dfs::Vector{Void}) = dfs
-function Base.vcat{T<:AbstractDataTable}(dfs::Vector{T})
-    isempty(dfs) && return DataTable()
-    coltyps, colnams, similars = _colinfo(dfs)
+Base.vcat(dts::Vector{Void}) = dts
```
Comment (tkelman) [inline src/abstractdatatable/abstractdatatable.jl:687]: is this method needed?

Comment (ararslan) [reply src/abstractdatatable/abstractdatatable.jl:687]: Not sure what it would be used for... plus this is piracy.

Comment (tkelman) [reply src/abstractdatatable/abstractdatatable.jl:687]: I get
```
julia> vcat(Vector{Void}())
0-element Array{Void,1}
```
just fine - maybe an ambiguity-breaker in old julia versions?

Comment (ararslan) [reply src/abstractdatatable/abstractdatatable.jl:687]: ¯\\\_(ツ)\_/¯

Comment (tkelman) [reply src/abstractdatatable/abstractdatatable.jl:687]: should we try deleting it?

Comment (cjprybol) [reply src/abstractdatatable/abstractdatatable.jl:687]: https://github.com/JuliaData/DataTables.jl/pull/18

it passed tests for me on v0.5

**Pull request merged**